modules/ROOT/pages/_partials/include-insights-charts.adoc

```diff
@@ -0,0 +1,386 @@
+Young Gen GC Duration::
+Amount of time taken for young generation Java garbage collection
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Heap Committed::
+Amount of physical memory committed to the JVM
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Heap Usage::
+Amount of physical memory used
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Thread Count::
+//TODO: QUESTION: threads that are available or used??? (SATHISH)
+Number of threads for simultaneous requests
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Committed Virtual Memory::
+Amount of memory (in MB) committed to the JVM
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+CPU Utilization::
+CPU work as a percentage of total CPU time
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+//end::app-detail-jvm[]
+//------------------------------------------------------//
+
+//========================//
+//API DETAILS: ALL METRICS//
+//========================//
+//
+// API DETAILS: Header-level metrics
+//-------------------------
+//tag::api-detail-header[]
+
+//TODO: Api -> API (SATHISH)
+Type::
+API implementation type (*API*)
+
+//TODO: verify for Insights; info is from API Manager docs (SATHISH)
+Status::
+Status code that indicates whether the governance policies and SLA tiers are functional (*Active*) or not
++
+For all codes, see information on xref:api-manager::latest-overview-concept.adoc#what-api-manager-looks-like[tracking registration status] in the API Manager UI documentation.
+
+//TODO: Url -> URL (SATHISH)
+Implementation URL:: 
+URL of deployed API integration
+
+//TODO: Runtime version -> Runtime Version (SATHISH)
+//TODO: issue: value can be empty; SEE BOOKS API IN STGX (SATHISH)
+//(SATHISH)
+Runtime Version::
+Mule Gateway or Flex Gateway runtime version used by the API implementation 
+
+//TODO: issue: value can be empty; SEE BOOKS API IN STGX 
+API Instance ID::
+Instance identifier for an API implementation incorporated into a Flex Gateway or API Gateway runtime
+
+//TODO: version -> Version (SATHISH)
+API Version::
+Version of the API implementation
++
+For more information, see xref:api-manager::api-instance-landing-page.adoc[].
+//end::api-detail-header[]
+//-----------------------//
+//
+//-------------------------
+// API DETAILS: Overview tab
+//-------------------------
+//tag::api-detail-overview[]
+
+//shared Overview metric
+include::partial$include-insights-charts.adoc[tag="average-latency"]
+
+//shared Overview metric
+include::partial$include-insights-charts.adoc[tag="p95-latency"]
+
+//shared Overview metric
+include::partial$include-insights-charts.adoc[tag="total-requests"]
+
+//TODO: issue: 2 charts named Policy Violations
+//TODO: Policy Violations -> Policy Violation Count (or similar) (SATHISH)
+Policy Violations::
+Number of requests blocked because of API policy violations
```

Comment:
```suggestion
//TODO: issue: 2 charts named Policy Violations
//TODO: Policy Violations -> Policy Violation Count (or similar) (SATHISH)
Policy Violations::
Number of requests blocked because of API policy violations
```
@sathishpaulleo

Reply:
Ask API Manager folks. See Slack